### ISSUE TITLE
Having trouble when profiling MaxViT

### ISSUE THREAD

**Issue by MurazeKrinas** [ThanatosShinji/onnx-tool#55]
Dear author,

I'm having a trouble when profiling the MaxViT model ([arxiv.org/pdf/2204.01697v4.pdf](https://arxiv.org/pdf/2204.01697v4.pdf)). I used timm library for importing (maxvit_base_tf_224) and torch.onnx (Opset version 17) to perform ONNX converting. Futhermore, I ultilized Polygraphy provided by TensorRT (https://github.com/NVIDIA/TensorRT/tree/main/tools/Polygraphy/examples/cli/surgeon/02_folding_constants) to do constant folding the graph. The error has been described in the image and the ONNX file is attached below.

Hope to hear from you soon. I'm Vietnamese so maybe the grammar is not so good! I hope you understand.

MaxViT-B_folded.onnx: https://drive.google.com/file/d/1mLlJ_0IFs1RdVFbFfwB2iDyI8HDZxCun/view?usp=sharing

![image](https://github.com/ThanatosShinji/onnx-tool/assets/88036898/3e84a82e-d663-4594-8e99-c9301199f4e6)

**Comment by ThanatosShinji** [comment]
It seems your graph is not correct. '/stem/conv1/Cast_9_output_0' This padding value should be [1,1, ..], but your graph gives an all-zero tensor.

**Comment by ThanatosShinji** [comment]
It's a bug. The root cause is that two Clip nodes '/stem/conv1/Clip_1' and '/stem/conv1/Clip' have 3 input tensors. But the last tensor has an empty tensor name which should set no_max to clip.

**Comment by ThanatosShinji** [comment]
The fix commit has been pushed to the main branch.

**Comment by MurazeKrinas** [comment]
Hello Thanatos Shinji,

Thank you for taking the time to fix the error, I really appreciate it! Now I can profile the MaxViT model and the parameters look very good! Regarding your opinion, I used _onnxruntime_ to check as well as inference the model, everything works fine, otherwise as I mentioned, everything is imported from reputable libraries such as _torch_, _timm_, etc. So I think they built the architecture like that for a reason. I will research more and inform you if possible.

Currently, I have just discovered 2 other models with bugs when profiling, should I create a new issue or update in this issues's comment?
